apps/storybook/panda.config.ts

```diff
@@ -1,34 +1,12 @@
 import { defineConfig } from "@pandacss/dev";
 
-import {
-  conditions,
-  globalCss,
-  recipes,
-  semanticTokens,
-  slotRecipes,
-  textStyles,
-  tokens,
-} from "@tailor-platform/design-systems/client";
+import { buildPandaConfig } from "./node_modules/@tailor-platform/dev-config/pandacss";
```

Comment:
細かいですがここは以下で大丈夫かと思います。
```suggestion
import { buildPandaConfig } from "@tailor-platform/dev-config/pandacss";
```

Reply:
OKです。一旦ここはこのままでいきましょう。